src/nvim/tui/tui.c

```diff
@@ -51,10 +52,15 @@
 
 // Per the commentary in terminfo, only a minus sign is a true suffix
 // separator.
-#define TERMINAL_FAMILY(term, prefix) ((term) \
-    && strlen(term) >= (sizeof(prefix) - 1) \
-    && 0 == memcmp((term), (prefix), sizeof(prefix) - 1) \
-    && ('\0' == (term)[sizeof(prefix) - 1] || '-' == (term)[sizeof(prefix) - 1]))
+bool terminfo_is_term_family(const char *term, const char *family)
```

Comment:
It looks like this function is only used in. Until we're using it anywhere else, I'd mark it `static`.